WordPress/WordPressUITests/Tests/LoginTests.swift

```diff
@@ -85,11 +85,6 @@ class LoginTests: XCTestCase {
             // Login flow returns MySites modal, which needs to be closed.
             .closeModal()
 
-            // TODO: rewrite logoutIfNeeded() to handle logging out of a self-hosted site and then WordPress.com account.
```

Comment:
This was causing tests to fail locally, and since we are no longer using `logoutIfNeeded()` we don't really this anymore.

Reply:
Sounds good.

Out of curiosity, why are the tests failing locally?

Reply:
One quick correction from my side - it was actually one test, not many. 

When running the `testAddSelfHostedSiteAfterWPcomLogin` test, after removing the self-hosted site, the test would return to My Site Screen. Because of the scroll from `removeSelfHostedSite()` method, the [first element](https://github.com/wordpress-mobile/WordPress-iOS/blob/0de35b1c4375b2ab29582367e7b025f6308f4ef2/WordPress/UITestsFoundation/Screens/MySiteScreen.swift#L73) that My Site Screen was looking for was hidden from view. Initially thought of adding a scroll so that the element can be located but since this is actually not needed anymore, removing it seems to be a better way.